plugin/storage/es/spanstore/reader_test.go

```diff
@@ -82,23 +85,43 @@ var exampleESSpan = []byte(
 	   }
 	}`)
 
+type closed func() error
+
 type spanReaderTest struct {
 	client    *mocks.Client
 	logger    *zap.Logger
 	logBuffer *testutils.Buffer
+	exporter  *tracetest.InMemoryExporter
+	closer    closed
```

Comment:
this isn't any better, I can already tell from the field name that it "closes something", just not what it closes. Adding a type like that does not help. But naming the field `tracerCloser` make is obvious, without any additional type.